packages/trader/src/Modules/Trading/Containers/trade-params-mobile.tsx

```diff
@@ -20,20 +74,21 @@ const DEFAULT_DURATION = Object.freeze({
     d: 1,
 });
 
-const reducer = (state, payload) => {
+const reducer = (state: TReducer, payload: Partial<TReducer>) => {
     return {
         ...state,
         ...payload,
     };
 };
 
-const makeGetDefaultDuration = (trade_duration, trade_duration_unit) => duration_unit =>
-    trade_duration_unit === duration_unit ? trade_duration : DEFAULT_DURATION[duration_unit];
+const makeGetDefaultDuration = (trade_duration: number, trade_duration_unit: string) => (duration_unit: string) =>
+    trade_duration_unit === duration_unit
+        ? trade_duration
+        : DEFAULT_DURATION[duration_unit as keyof typeof DEFAULT_DURATION];
 
-const TradeParamsModal = observer(({ is_open, toggleModal }) => {
-    const { client, ui } = useStore();
+const TradeParamsModal = observer(({ is_open, toggleModal }: TTradeParamsModal) => {
+    const { client } = useStore();
     const { currency } = client;
-    const { enableApp, disableApp } = ui;
```

Comment:
`enableApp` and `disableApp` are not been used in `<Modal />` component, it's not expecting them. On localhost was working fine without them, so I decided to delete them.